src/components/CreateNewVM.vue

```diff
@@ -79,6 +79,10 @@
             <q-input v-model="newProgram.entrypoint" label="Your entrypoint"
             stack-label standout class="q-my-sm" />
         </div>
+        <div class="col-12">
+            <q-input v-model="newProgram.metadata.name" label="Name your VM"
```

Comment:
What happends if `newProgram.metadata === null` ?

Reply:
Only the hash will be displayed

Here an example:

![Capture d’écran 2022-12-23 à 11 59 32](https://user-images.githubusercontent.com/17054452/209324512-4fbba201-f940-4000-a07b-f5a76a0f097b.png)